install/k8s/install.sh

```diff
@@ -234,8 +234,8 @@ if [ "$tls_cert" = "" ]; then
 	tls_cert=./local_certs/cert.pem
 	tls_key=./local_certs/local.key
 fi
-cp $tls_cert /var/contiv/auth_proxy_cert.pem
-cp $tls_key /var/contiv/auth_proxy_key.pem
+cp -u $tls_cert /var/contiv/auth_proxy_cert.pem
```

Comment:
This is one of the issues preventing it, but there is more work to be done as well which will come in separate PRs